ui-tests/tests/deploy_sql.test.ts

```diff
@@ -0,0 +1,13 @@
+import { test } from '@jupyterlab/galata';
+import { expect } from '@playwright/test';
+
+test('test deploy notebook', async ({ page }) => {
```

Comment:
this says "test deploy notebook" but seems like it's testing the "SQL format" feature?

Reply:
addressed